code/modules/reagents/reagent_containers/blood_pack.dm

```diff
@@ -43,9 +43,9 @@
 		if(51 to INFINITY)
 			icon_state = "full"
 
-/obj/item/weapon/reagent_containers/blood/random/New()
+/obj/item/weapon/reagent_containers/blood/random/Initialize()
 	blood_type = pick("A+", "A-", "B+", "B-", "O+", "O-", "L")
-	..()
```

Comment:
This was running code before parent New() that would now happen after New().

Does this matter in this case?